docs/de/docs/fastapi-people.md

```diff
@@ -0,0 +1,171 @@
+# FastAPI Personen
+
+FastAPI hat eine erstaunliche Community, die Menschen mit unterschiedlichstem Hintergrund willkommen heißt.
+
+## Erfinder - Maintainer
+
+Hey! 👋
+
+Das bin ich:
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.maintainers %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Answers: {{ user.answers }}</div><div class="count">Pull Requests: {{ user.prs }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Ich bin der Erfinder und Betreuer von **FastAPI**. Sie können mehr darüber in [Help FastAPI - Get Help - Connect with the author](help-fastapi.md#connect-with-the-author){.internal-link target=_blank} erfahren.
+
+...Aber hier möchte ich Ihnen die Community zeigen.
+
+---
+
+**FastAPI** erhält eine Menge Unterstützung aus der Community. Und ich möchte ihre Leistungen hervorheben.
+
+Das sind die Personen, die:
+
+* [Anderen bei Problemen (Fragen) in GitHub helfen](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
+* [Pull Requests erstellen](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
+* Pull Requests überprüfen, [besonders wichtig für Übersetzungen](contributing.md#translations){.internal-link target=_blank}.
+
+Eine Runde Applaus für sie. 👏 🙇
+
+## Aktivste Benutzer im letzten Monat
+
+Dies sind die Benutzer, die im letzten Monat am meisten [anderen mit Problemen (Fragen) auf Github](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} geholfen haben. ☕
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.last_month_active %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
```

Comment:
```suggestion
<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Antworten: {{ user.count }}</div></div>
```

This can be translated.